zuul.d/projects.yaml

```diff
@@ -662,15 +662,14 @@
     default-branch: devel
     templates:
       - system-required
-      - ansible-python-jobs
       - ansible-python3-jobs
       - publish-to-pypi
     check:
       jobs:
-        - ansible-tox-py27
+        - ansible-tox-py36
```

Comment:
by doing this, you'll drop py27 from all branches, is that what you'd like to do?

Reply:
no, would like to make it conditional - was checking out https://zuul-ci.org/docs/zuul/reference/job_def.html#attr-job.branches just now

Reply:
took a crack at configuring zuul to only run py2 jobs against specific branches

Reply:
You can actually do the following, for both check and gate pipelines.
```
jobs:
  - ansible-tox-py27:
      branches: ^stable/.*
  - ansible-tox-py36
```